perf/perf-producer.go

```diff
@@ -62,6 +63,8 @@ func newProducerCommand() *cobra.Command {
 		"Publish rate. Set to 0 to go unthrottled")
 	flags.IntVarP(&produceArgs.BatchingTimeMillis, "batching-time", "b", 1,
 		"Batching grouping time in millis")
+	flags.IntVarP(&produceArgs.BatchingMaxSize, "batching-max-size", "", 128,
+		"Max size of a batch (in KB)")
```

Comment:
In `pulsar-perf produce` about `--batch-max-messages` is abbreviated and the default value is as follows, can I consider keeping the two consistent?


``` 
   -bm, --batch-max-messages
       Maximum number of messages per batch
       Default: 1000
```

Reply:
This are 2 different settings. One is the number of messages in 1 batch and the other is the max size of the batch